policies/rules.md

```diff
@@ -138,7 +138,7 @@ Competition in this Hackathon officially begins on March 16th at 00:00 UTC and c
 
 Participants understand that this is a charitable event organized by volunteers with limited resources and time. As such, participants understand that a best effort will be made to ensure fairness and equality in the judging and awarding of prizes. Participants accept that prizes will be awarded at the discretion of the judges, sponsors, and organizers. By participating all participants agree to accept the decisions of the judges, sponsors and organizers, and waive all rights to pursue legal or other recourse against the event, organizers, judges, sponsors, partners, or other parties should a dispute arise.
 
-All participants, organizers, sponsors and contributors agree to follow [the CODEVID-19 Code of Conduct](code-of-conduct.html) and understand that failure to do so could result in consequences up to and including elimination and exclusion from competition, Slack, and other resources provided along with forfeiture of any prize winnings or consideration provided by CODEVID-19, the Dev Edmonton Society, and associated parties. Participants understand that enforcing and interpreting the Code of Conduct is done at the sole discretion of the CODEVID-19 organizing volunteers. Questions regarding the enforcement or interpretation of the Code of Conduct should be directed to conduct@codevid19.com.
+All participants, organizers, sponsors and contributors agree to follow the CODEVID-19 [Code of Conduct](code-of-conduct.html) and understand that failure to do so could result in consequences up to and including elimination and exclusion from competition, Slack, and other resources provided along with forfeiture of any prize winnings or consideration provided by CODEVID-19, the Dev Edmonton Society, and associated parties. Participants understand that enforcing and interpreting the [Code of Conduct](code-of-conduct.html) is done at the sole discretion of the CODEVID-19 organizing volunteers. Questions regarding the enforcement or interpretation of the [Code of Conduct](code-of-conduct.html) should be directed to [conduct@codevid19.com](mailto:conduct@codevid19.com).
```

Comment:
The only changes here were to add hyperlinks to the 2 other spots in this paragraph that refer to the CoC.